.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [ 18 ]
+        node-version: [ 22 ]
```

Comment:
Should we leave this lower so that we maintain compatibility with older node versions?

Reply:
Version 18 only has 6 months of maintenance support left (https://en.wikipedia.org/wiki/Node.js#Releases).